modules/app/src/main/java/org/locationtech/jtstest/testbuilder/io/shapefile/Shapefile.java

```diff
@@ -1,3 +1,23 @@
+/*
+ * Copyright (c) 2003 Open Source Geospatial Foundation, All rights reserved.
+ * 
+ * This program and the accompanying materials are made available under the terms
+ * of the OSGeo BSD License v1.0 available at:
+ *
+ * https://www.osgeo.org/sites/osgeo.org/files/Page/osgeo-bsd-license.txt
+ */
+/*
```

Comment:
My reasoning was that this code is primarily GeoTools, with mods by Vivid.  But can see the logic in date order as well.